python/pyarrow/tests/parquet/test_pandas.py

```diff
@@ -150,52 +144,22 @@ def test_pandas_parquet_2_0_roundtrip_read_pandas_no_index_written(
     tm.assert_frame_equal(df, df_read)
 
 
-# TODO(dataset) duplicate column selection actually gives duplicate columns now
```

Comment:
Do you know if this is actually tested elsewhere? (that you get duplicate columns)

Reply:
I think there is only one similar test using datasets
https://github.com/apache/arrow/blob/3c66491846a24f17014b31a22fafdda0229f881a/python/pyarrow/tests/test_dataset.py#L3472-L3473
which is same at the end but will still add another for `pq.read_table` which is now using only the new dataset API. Not sure if I have to roundtrip with pandas though?

Reply:
https://github.com/apache/arrow/pull/39112/commits/5b1f6eda7b8696989ff4487171770677912c58fb